include/emitc/Target/Cpp.h

```diff
@@ -86,6 +86,9 @@ struct CppEmitter {
   /// the array size.
   LogicalResult emitTupleType(ArrayRef<Type> types);
 
+  /// Emits an assignment for a variable which has been declared previously.
+  LogicalResult emitVariableAssignment(OpResult result);
+
   /// Emits a variable declaration for a result of an operation.
   LogicalResult emitVariableDeclaration(OpResult result,
                                         bool trailingSemicolon);
```

Comment:
Do we want to reintroduce the default `= true` here, if it is removed from `emitOperation`?

Reply:
I'd vote for no. Having to think about this for each operation is a good thing.

Reply:
I am fine with this :+1: